### PR TITLE
CIQUAL ingredients continued

### PR DESCRIPTION
**Pull request by aleene**
More Ciqual additions and ingredients repairs

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -66889,12 +66971,58 @@ fr:flétan
...
+wikipedia:en:https://en.wikipedia.org/wiki/Greenland_halibut
+description:de:Der Schwarze Heilbutt (Reinhardtius hippoglossoides) ist ein großer Plattfisch, der in kalten Meeren der nördlichen Hemisphäre lebt.
+description:en:The Greenland halibut or Greenland turbot (Reinhardtius hippoglossoides) belongs to the family Pleuronectidae.
+description:fr:Le flétan du Groenland (Reinhardtius hippoglossoides) est une espèce de poissons plats de la famille des Pleuronectidae. 
+description:it:L'halibut della Groenlandia (Reinhardtius hippoglossoides), noto commercialmente come ppoglosso nero o con il nome generico di halibut, è un pesce di mare della famiglia Pleuronectidae.
+description:nl:De Groenlandse heilbot of zwarte heilbot (Reinhardtius hippoglossoides) is een straalvinnige vis uit de familie van schollen (Pleuronectidae).
+
+<en:Greenland halibut
 en:Raw Greenland halibut
 fr:Flétan du Groënland cru, Flétan noir cru, Flétan commun cru
```
Comment (teolemon) [inline taxonomies/ingredients.txt:67005]:
```suggestion
en:Greenland halibut, Raw Greenland halibut
fr:Flétan du Groënland, Flétan noir cru, Flétan commun cru, Flétan du Groënland cru
```

**Comment by teolemon** [comment]
@stephanegigandet @aleene I'm going to solve the conflict and merge

**Review by teolemon — APPROVED** [review]

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -2823,7 +2823,7 @@ wikipedia:en:https://en.wikipedia.org/wiki/Glucono_delta-lactone
 # usage:fr:poudres à lever (glucono-delta-lactone, carbonate acide de sodium)
 
 
-# description:en:CALCIUM GLUCONATE is a mineral supplement and medication.
+# description:en:meCONATE is a mineral supplement and medication.
```
Comment (teolemon) [inline taxonomies/ingredients.txt:2826]: @aleene

**Review by teolemon — COMMENTED** [review]
A lot of disabled ingredients thanks to processing. Will it work ? Do we have sufficient test coverage ?

**Comment by aleene** [comment]
Yes the test coverage should be ok. We created the processing taxonomy, but never really applied it well. That is the work I am doing now with the disabling.

**Comment by teolemon** [comment]
Merged, thanks @aleene 👏

**Pull request merged**